examples/illustrations/existence_intervals/README.md

```diff
@@ -115,3 +115,101 @@ Some of the examples in the OWL-TIME specification include demonstrations of `ti
+
+* `gufo:Endurant` corresponds with UCO Issue 535's "Endurant".
+* `gufo:Event` corresponds with UCO Issue 535's "Perdurant".
+* `gufo:Situation` is left out of scope of this discussion.
+
+Two properties, "has begin point" and "has end point," assign the first and last instants that any concrete individual exists.  The properties have range `time:Instant`, inheriting the vocabulary and entailing the structures of OWL-Time as described above.
+
+None of the three subclasses of `gufo:ConcreteIndividual` specialize restrictions on those existence-bounding properties.  Hence, without loss of generality, we can review a `gufo:Endurant` and see the same time-related applicability for `gufo:Event`.
+
+This is an endurant (more specifically, a `gufo:Object`) with literal-valued instants describing the creation and termination timestamps TS0 and TS1:
+
+```json
+[
+    {
+        "@id": "kb:Object-bee97e78-8e5e-4ffe-9c2b-28e960f84bbc",
+        "@type": "gufo:Object",
+        "rdfs:label": "T",
+        "rdfs:comment": "A thing with a time-bounded existence.",
+        "gufo:hasBeginPointInXSDDateTimeStamp": {
+            "@type": "xsd:dateTimeStamp",
+            "@value": "2020-01-02T03:04:05.6789Z"
+        },
+        "gufo:hasEndPointInXSDDateTimeStamp": {
+            "@type": "xsd:dateTimeStamp",
+            "@value": "2021-02-03T04:05:06.7890Z"
+        }
+    }
+]
+```
+
+This is the same endurant with reified instants housing the beginning and end:
+
+```json
+[
+    {
+        "@id": "kb:Object-bee97e78-8e5e-4ffe-9c2b-28e960f84bbc",
+        "rdfs:label": "T",
+        "gufo:hasBeginPoint": {
+            "@id": "kb:Instant-1a250c7b-19c5-4ec4-bed2-d05c9d438e9d",
+            "@type": "time:Instant",
+            "rdfs:label": "T0",
+            "time:inXSDDateTimeStamp": {
+                "@type": "xsd:dateTimeStamp",
+                "@value": "2020-01-02T03:04:05.6789Z"
+            }
+        },
+        "gufo:hasEndPoint": {
+            "@id": "kb:Instant-bbd8741a-4af8-4358-802b-a6fb8a4cf7dc",
+            "@type": "time:Instant",
+            "rdfs:label": "T1",
+            "time:inXSDDateTimeStamp": {
+                "@type": "xsd:dateTimeStamp",
+                "@value": "2021-02-03T04:05:06.7890Z"
+            }
+        }
+    }
+]
+```
+
+Intervals of existence do not appear to be defined directly in gUFO.  Also, since only the first and last instant of a concrete individual are specified, it is also not explicit in gUFO whether an individual's interval of existence needs to be continuous.  For instance, a `gufo:Situation` may be seen to hold for a period of time (such as a person being a student in a `gufo:TemporaryInstantiationSituation`), then not hold (e.g., when the person graduates from a school), then hold again (e.g., when the person later enrolls in another school).
+
```

Comment:
I'll make a footnote of this.  I think explaining the other individual gets a little too big for a parenthetical remark.

Reply:
Footnote added.  Please let me know if you think it needs further adjusting, or if it's better to just omit `Situation` discussion entirely.

Reply:
Nit-picking here would be that it is, imo, not necessary to elaborate on how it is done, just the essentials that each distinct enrollment is represented as distinct `gufo:TemporaryInstantiationSituation`s. As in: don't make it more complicated than it needs to be. But again, this is nit-picking because the purpose has been achieved already.